Source/WebCore/WebCore.xcodeproj/project.pbxproj

```diff
@@ -6013,7 +6012,6 @@
 			dstPath = "";
 			dstSubfolderSpec = 16;
 			files = (
-				DDB04F3A278E5539008D3678 /* libWTF.a in Product Dependencies */,
```

Comment:
Ditto above, we'd need to add 
```
$(BUILT_PRODUCTS_DIR)/usr/local/include/wtf/Scripts/generate-unified-source-bundles.rb
```
to the inputs of Generate Unified Sources in order to remove this.

Reply:
This too.